modules/reactor/test/tests/meshgenerators/peripheral_ring_mesh_generator/ring_quadratic.i

```diff
@@ -0,0 +1,33 @@
+[Mesh]
+  [pccmg]
+      type = PolygonConcentricCircleMeshGenerator
+      num_sides = 8
+      num_sectors_per_side = '4 4 4 4 4 4 4 4'
+      background_intervals = 2
+      polygon_size = 5.0
+
+      preserve_volumes = on
+
+      tri_element_type = TRI6
+      quad_element_type = QUAD8
```

Comment:
Why `QUAD8` instead of `QUAD9`? Just curious.

Reply:
this PR adds quad9 for the periphery. for the mesh inside, quad8 will use less memory and is just as compatible with the boundary